__tests__/pages/curriculum.test.js

```diff
@@ -139,4 +142,20 @@ describe('Curriculum Page', () => {
 
     await wait(() => expect(container).toMatchSnapshot())
   })
+  test('Should return to login page if unauthorized', async () => {
+    const mocks = [
+      {
+        request: { query: GET_APP },
+        result: {
+          errors: [new GraphQLError("Cannot read property 'id' of null")]
+        }
+      }
+    ]
+    const { container } = render(
+      <MockedProvider mocks={mocks} addTypename={false}>
+        <Curriculum />
+      </MockedProvider>
+    )
+    await wait(() => expect(container).toMatchSnapshot())
```

Comment:
`wait` is deprecated, and `waitFor` should be used instead

https://testing-library.com/docs/dom-testing-library/api-async/#waitfor

Reply:
`waitFor` requires more modern version of testing-library (c0d3-app uses ^9.4.0). Upgrading it to the latest 11something version breaks all snapshots, so it probably needs to be a separate pr for this.

Reply:
update your fork, PR #464 was merged and updated a lot of deps

Reply:
Updating local branch causes some weird errors in tests `TestingLibraryElementError: Unable to find role="heading"` even if I completely override local files when merging. Quick search gives this [issue](https://github.com/testing-library/dom-testing-library/issues/578) but according to it this problem should be fixed in modern versions of testing library. 

I'll try to figure it out today but closing this pr and opening the new one from synced master could be faster in the end.